admin_manual/enterprise_clients/publishing_android_app.rst

```diff
@@ -0,0 +1,333 @@
+to the Google Play Store.
+
+Google's process for uploading apps is fairly streamlined, and the most 
+time-consuming task is creating all the required graphics. After registering, 
+you'll see the welcome screen for the Google Dev Console. Click **Publish an 
+Android app on Google Play**. 
+
+.. image:: ../images/android_custom_6.png
+
+This opens the **Add New Application** screen. Click the **Prepare Store 
+Listing** button. (Note that as you navigate the various screens, you can click 
+the Save Draft button to preserve your changes.)
+
+.. image:: ../images/android_custom_7.png
+
+On the next screen, enter your product description.
+
+.. image:: ../images/android_custom_8.png
+
+Then you'll have to upload a batch of graphics in various sizes for the 
+**Graphic Assets** section, like these images for a smartphone and seven-inch 
+tablet. You are required to upload at least two images.
+
+.. image:: ../images/android_custom_9.png
+
+You must also upload a 512x512-pixel logo, and a 1024x500 banner.
+
+.. image:: ../images/android_custom_10.png
+
+Now choose the store categories for your app.
+
+.. image:: ../images/android_custom_11.png
+
+Then enter your contact information, which will be visible on your store 
+listing.
+
+.. image:: ../images/android_custom_12.png
+
+On the next line you may optionally link to your privacy policy. It is 
+recommended to have a privacy policy.
+
+When you're finished with the **Store Listing** page, go to the **Pricing and 
+Distribution** page. You may make this a paid or free app. You cannot convert a 
+free app to paid. You may convert a paid app to free, but then you can't 
+convert it back to paid. You'll have numerous options for paid apps, such as 
+Android Wear, Android TV, and various Google marketing tie-ins, and many more. 
+
+For now let's make this a free app, so click the Free button and select the 
+countries you want to distribute it in.
+
+.. image:: ../images/android_custom_13.png
+
+Now you may upload your app.
+ 
+Uploading to Google Play Store
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now you can upload your app to your Google Play Store page. Go to the **APK** 
+page and click **Upload your first APK to Production**. You don't need a license 
+key for a free app.
+
+.. image:: ../images/android_custom_14.png
+
+Drag-and-drop, or browse to select your app.
+
+.. image:: ../images/android_custom_15.png
+
+A successful upload looks like this:
+
+.. image:: ../images/android_custom_20.png
+
+Your app is not yet published, but only uploaded to your account. There is one 
+more step to take before you can publish, and that is to go back to the 
+**Pricing & Distribution** page and fill out the **Consent** section. 
+
+.. image:: ../images/android_custom_21.png
+
+Click the Save Draft button, and if you followed all the required steps you 
+should now see a **Publish App** button. 
+
+.. image:: ../images/android_custom_22.png
+
+It will not be published immediately, but after review by Google.
+
```

Comment:
Not sure whether it´s needed to be added, Google review is not as deep as the iOS one they don´t check the code (ftm). Google usually takes a couple of hours and Apple 1-2 weeks.